app/views/pages/home.html.erb

```diff
@@ -55,6 +55,73 @@
+                <path d="M25.3333 13.1667H31.6666L36.4166 17.9167V25.8334H25.3333V13.1667Z" stroke="#415262" stroke-width="3" stroke-linecap="round" stroke-linejoin="round" />
+                <path d="M8.70833 33.7499C10.8945 33.7499 12.6667 31.9777 12.6667 29.7916C12.6667 27.6055 10.8945 25.8333 8.70833 25.8333C6.52221 25.8333 4.75 27.6055 4.75 29.7916C4.75 31.9777 6.52221 33.7499 8.70833 33.7499Z" stroke="#415262" stroke-width="3" stroke-linecap="round" stroke-linejoin="round" />
+                <path d="M29.2916 33.7499C31.4777 33.7499 33.2499 31.9777 33.2499 29.7916C33.2499 27.6055 31.4777 25.8333 29.2916 25.8333C27.1055 25.8333 25.3333 27.6055 25.3333 29.7916C25.3333 31.9777 27.1055 33.7499 29.2916 33.7499Z" stroke="#415262" stroke-width="3" stroke-linecap="round" stroke-linejoin="round" />
+              </svg>
+            </div>
+            <h3 class="text-xl font-bold text-primary-700">National Guidelines</h3>
+            <p class="font-bold text-accent-100"><%= OccupationStandard.by_national_standard_type(:guideline_standard).count %> Apprenticeships</p>
+          </div>
+        <% end %>
+        <%= link_to occupation_standards_path(national_standard_type: {occupational_framework: 2}) do %>
+          <div class="p-6 hover:bg-white rounded-lg hover:shadow-xl">
+            <div class="mx-auto flex justify-center items-center mb-4 w-20 h-20 rounded-full bg-accent-200">
+              <svg width="38" height="39" viewBox="0 0 38 39" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path d="M25.3333 5.25H1.58325V25.8333H25.3333V5.25Z" stroke="#415262" stroke-width="3" stroke-linecap="round" stroke-linejoin="round" />
+                <path d="M25.3333 13.1667H31.6666L36.4166 17.9167V25.8334H25.3333V13.1667Z" stroke="#415262" stroke-width="3" stroke-linecap="round" stroke-linejoin="round" />
+                <path d="M8.70833 33.7499C10.8945 33.7499 12.6667 31.9777 12.6667 29.7916C12.6667 27.6055 10.8945 25.8333 8.70833 25.8333C6.52221 25.8333 4.75 27.6055 4.75 29.7916C4.75 31.9777 6.52221 33.7499 8.70833 33.7499Z" stroke="#415262" stroke-width="3" stroke-linecap="round" stroke-linejoin="round" />
+                <path d="M29.2916 33.7499C31.4777 33.7499 33.2499 31.9777 33.2499 29.7916C33.2499 27.6055 31.4777 25.8333 29.2916 25.8333C27.1055 25.8333 25.3333 27.6055 25.3333 29.7916C25.3333 31.9777 27.1055 33.7499 29.2916 33.7499Z" stroke="#415262" stroke-width="3" stroke-linecap="round" stroke-linejoin="round" />
+              </svg>
+            </div>
+            <h3 class="text-xl font-bold text-primary-700">Occupational Frameworks</h3>
+            <p class="font-bold text-accent-100"><%= OccupationStandard.by_national_standard_type(:occupational_framework).count %> Apprenticeships</p>
+          </div>
+        <% end %>
+        <%= link_to occupation_standards_path(ojt_type: {time: 0}) do %>
```

Comment:
This should be a "1" if we are trying to add a path to the time_based standards:

```suggestion
        <%= link_to occupation_standards_path(ojt_type: {time: 1}) do %>
```

Reply:
This is how time is `enum ojt_type: [:time, :competency, :hybrid], _suffix: :based`
I copied the structure from the old home page. Does the number not equate to the index?

Reply:
If you use the search filter on the Occupation Standards page and check "Hybrid" and "Time", then the parameters come through to the controller like so:

```
#<ActionController::Parameters {"q"=>"", "ojt_type"=>{"hybrid"=>"1", "time"=>"1"}, "state_id"=>"", "controller"=>"occupation_standards", "action"=>"index"} permitted: false>
```

We want to pass that "1" value here for the time based standards.

Reply:
I've changed it to 1